app/components/workbench/Preview.tsx

```
@@ -56,28 +77,148 @@
   useEffect(() => {
     if (previews.length > 1 && !hasSelectedPreview.current) {
       const minPortIndex = previews.reduce(findMinPortIndex, 0);
-
       setActivePreviewIndex(minPortIndex);
     }
-  }, [previews]);
+  }, [previews, findMinPortIndex]);
 
   const reloadPreview = () => {
     if (iframeRef.current) {
       iframeRef.current.src = iframeRef.current.src;
```

Biome (1.9.4)

[error] src is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)